tests/tests_integration/test_api/test_simulators/conftest.py

```diff
@@ -3,20 +3,26 @@
 import time
 from collections.abc import Iterator
 from pathlib import Path
+from typing import Any
```

Comment:
AS before, Dag does not fully approve!

Reply:
it's a common approach in the SDK, Dag is looking for better ways, most likely it will require some rewrite of the internals